api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/TaskTranslatorImplTest.java

```diff
@@ -12,6 +12,7 @@
 import static org.hamcrest.Matchers.notNullValue;
 import static org.junit.Assert.assertThat;
 import static org.hamcrest.Matchers.equalTo;
```

Comment:
Can we organize these imports so all the hamcrest one's are together followed by the junit one?